src/NServiceBus.Core/Licensing/Browser.cs

```diff
@@ -39,32 +38,4 @@ public static bool TryOpen(string url)
             return true;
         }
     }
-}
-#else
-namespace NServiceBus
-{
-    using System.Diagnostics;
-
-    static class Browser
```

Comment:
Where did this class go?

Reply:
No where, it's still there. There used to be two different implementations of this class, but now there is just one.